src/corelibs/U2Gui/src/util/GObjectComboBoxController.cpp

```diff
@@ -79,7 +79,10 @@ void GObjectComboBoxController::addDocumentObjects(Document* d) {
     // checks whether you need to add a new annotations table
     QString docUrl = settings.relationFilter.ref.docUrl;
     if (d->getURLString() == docUrl) {
-        connect(d->getObjectById(settings.relationFilter.ref.entityRef.entityId), SIGNAL(si_lockedStateChanged()), SLOT(sl_lockedStateChanged()));
+        GObject* seqObj = d->getObjectById(settings.relationFilter.ref.entityRef.entityId);
+        CHECK(seqObj != nullptr, );
```

Comment:
If you return here you disable a bunch of code below that does not depend on presence of the sequence object.
Examples:
```QList<GObject*> listAnnotations = d->findGObjectByType(GObjectTypes::ANNOTATION_TABLE)```
``` foreach (GObject* obj, d->getObjects()) {```

Reply:
The correct rewrite of this method should remove the top-level 'if' block code completely (which is an old workaround or even bug) and use a separate explicit API to add custom options into the combo-box. 
In the case of sequence view it will be a ''Create-New-Feature-Table-In-The-Sequence-Object' option. This option should be added only if the format supports writable features.

This is a much bigger task than the problem in this issue and it will affect (require a review) of all places this combo controller is used.

To fix only this issue I'm asking not changing behavior of this method: because the old and the new code are not equal and review of all side effects is out of scope of this PR.

Why the behaviors are not equal:
Sequence object may be null, but there may still be annotations in the matched document (GenBank with no sequences but only annotations) and the nested `if (!hasAnnotationTable)` won't work: the execution will proceed to the loop at the end of method which contains the meaningful code.